spec/features/sending_invitations_spec.rb

```diff
@@ -22,7 +18,7 @@ def fill_out_all_fields
 
   scenario 'not filling out all fields' do
     fill_out_all_fields
-    fill_out_placeholder 'e-mail respondent', ''
+    fill_in 'respondentEmail', with: ''
```

Comment:
Ik vind het persoonlijk wel zo fijn in een featurespec om zoveel mogelijk human-readable tekst te hebben.

Als je `for`-attribuut aan de labels toevoegd(wat ook acessibility voor blinden ten goede komt), kun je `fill_in 'e-mail respondent', with: ''` doen.